### PR TITLE
release: 0.43.0

### PR DESCRIPTION
**Pull request by overcat**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ As this project is pre 1.0, breaking changes may happen for minor version bumps.
 
 ## Pending
 
+## 0.43.0
```
Comment (sreuland) [inline CHANGELOG.md:7]: do you intend to get the https://github.com/stellar/java-stellar-sdk/pull/566/  into this release also?

Comment (overcat) [reply CHANGELOG.md:7]: Yes!

**Comment by overcat** [comment]
Hi @sreuland, can you approve the PR?

**Comment by overcat** [comment]
We should release this version before Protocol 20 is activated on the mainnet.

**Review by JakeUrban — APPROVED** [review]

**Pull request merged**